ztp/siteconfig-generator/siteConfig/siteConfig.go

```diff
@@ -20,6 +20,8 @@ const Standard = "standard"
 const Master = "master"
 const ZtpAnnotation = "ran.openshift.io/ztp-gitops-generated"
 const ZtpAnnotationDefaultValue = "{}"
+const UnsetStringValue = "__unset_value__"
```

Comment:
I've never been a big fan of this convention... I'd rather use a `*string` and have `nil` represent "not set"...

That said, I know we use this elsewhere in the codebase, so I suppose it can stay :(

Reply:
flag.String needs to set a default value and it's a string type which cannot be the value of a nil pointer. I'd rather to keep it as "__unset_value__" to differentiate between "flag is set with an empty string" and "flag is unset" if needed